%ProgramData%/Microsoft/Windows/UUS/State/_active.uusver

```diff
@@ -0,0 +1 @@
+1305.2405.14022.0
```

Comment:
Looks like a system file of yours smuggled itself into our repository. 🥷🏻 please remove!

Reply:
done